hermes_base/settings.py

```diff
@@ -233,6 +233,11 @@
             'GROUP_ID': os.getenv('SCIMMA_AUTH_USERNAME', '') + '-' + os.getenv('HOPSKOTCH_GROUP_ID', 'hermes-dev'),
             'TOPIC_HANDLERS': {
                 'hermes.test': 'hermes.alertstream_handlers.ingest_from_hop.handle_hermes_message',
+                'hermes.astrometry': 'hermes.alertstream_handlers.ingest_from_hop.handle_hermes_message',
+                'hermes.discovery': 'hermes.alertstream_handlers.ingest_from_hop.handle_hermes_message',
+                'hermes.message': 'hermes.alertstream_handlers.ingest_from_hop.handle_hermes_message',
+                'hermes.photometry': 'hermes.alertstream_handlers.ingest_from_hop.handle_hermes_message',
+                'hermes.spectroscopy': 'hermes.alertstream_handlers.ingest_from_hop.handle_hermes_message',
```

Comment:
this should just use the hermes.* functionality I added a while back right? Since they all have the same handler...

Reply:
and also, when using something.*, you must have a handler for just plain "*" (everything else catch-all) defined. That catch-all handler could just be a logger or noop though - i think it's just needed to force the alertstream to get the full list of topics...

Reply:
Oh. I didn't know about the plain "*" requirement. (I think I had tried to use `hermes.*` shortly after you implemented that, but it didn't seem to work (hence this commit). I'll try to update it to use `hermes.*`